app/build.gradle

```diff
@@ -152,6 +152,8 @@ dependencies {
 
     implementation libraries.livedataKtx
 
+    implementation 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Shouldn't this go to dependencies.gradle?

Reply:
Yes, it should 👍